pkgs/applications/video/olive-editor/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qt5, frei0r, opencolorio, ffmpeg-full  }:
+
+stdenv.mkDerivation
+{
+  pname = "olive-editor";
+
+  version = "2019-03-18";
+
+  src = fetchFromGitHub {
+    owner = "olive-editor";
+    repo = "olive";
+    rev = "d153e4d7471122987098087a203323003ae8a128";
+    sha256 = "1gxpz6jg6dqjmvqkk0m9g4pz30lklwx51d73gka404a3w1j5wna6";
+  };
+
+  nativeBuildInputs = [ 
+    pkgconfig 
+    which 
+    qt5.qmake
+  ];
+
+  buildInputs = [
```

Comment:
Provides [hicolor icons](https://github.com/olive-editor/olive/blob/54c03efd5481fb596f7d0ecca3badab6376fe16e/olive.pro#L372) so you should add `hicolor-icon-theme`.